### PR TITLE
Add CrowdStrike Falcon sensor

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request:
- Adds the CrowdStrike Falcon sensor to the AMI.
- Gets rid of some odds and ends left over from when we used to include Tanium and Tenable in this AMI.

See also cisagov/ansible-role-crowdstrike#5.

## 💭 Motivation and context ##

- We must install the CrowdStrike Falcon sensor to maintain the COOL ATO.
- Leftover code should be removed, since it's unnecessary code.  Deleted code is code that cannot contain bugs.

## 🧪 Testing ##

All automated tests pass.  I also built a new OpenVPN AMI with these changes for our staging COOL environment and verified that:
- OpenVPN still functions.
- The CrowdStrike Falcon sensor has the correct customer ID and tags and starts up successfully.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.
- [x] Build and test a staging AMI that contains the changes from this PR.

## ✅ Pre-merge checklist ##

- [x] Revert to the default branch of [cisagov/ansible-role-crowdstrike](https://github.com/cisagov/ansible-role-crowdstrike) once cisagov/ansible-role-crowdstrike#5 is merged.
- [x] Revert to the default branch of [cisagov/ansible-role-openvpn](https://github.com/cisagov/ansible-role-openvpn) once cisagov/ansible-role-openvpn#73 is merged.
- [x] Finalize version.

## ✅ Post-merge checklist ##

- [x] Create a release.

### CODE REVIEW

**Review by jasonodoom — APPROVED** [review]

File: src/cdm.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+- hosts: all
+  name: Configure for the CISA CDM environment
+  vars:
+    ports:
+      # CrowdStrike
+      - direction: out
+        port: 443
+        proto: tcp
```
Comment (mcdonnnj) [inline src/cdm.yml]: Did we want to reposition this to be more alphabetically sorted?

**Review by mcdonnnj — APPROVED** [review]
Seems straightforward enough. One comment for your consideration.

File: src/cdm.yml (hunk 1 above)
Comment (jsf9k) [reply src/cdm.yml]: Please see commit 4ccac31.

**Review by dav3r — APPROVED** [review]
It looks like this PR also removes some leftover bits related to CDM Tanium and Nessus.  Can you please mention that in the PR description?

**Comment by jsf9k** [comment]
> It looks like this PR also removes some leftover bits related to CDM Tanium and Nessus. Can you please mention that in the PR description?

Good call.  Description updated.

File: src/cdm.yml (hunk 1 above)
Comment (jsf9k) [reply src/cdm.yml]: Please see commit 4ccac31.

**Review by dav3r — APPROVED** [review]
👍

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔

**Comment by jsf9k** [comment]
> LGTM heavy_check_mark

:open_mouth:

**Pull request merged**